docs/content/en/docs/installation/helm/_index.md

```diff
@@ -5,7 +5,31 @@ weight: 15
 ---
 
 The Aiven Operator for Kubernetes can be installed via [Helm](https://helm.sh/). 
-Please follow The instruction in the [aiven-charts](https://github.com/aiven/aiven-charts) GitHub repository.
+
+First add the [aiven helm repository](https://github.com/aiven/aiven-charts):
+
+```bash
+helm repo add aiven https://aiven.github.io/aiven-charts && helm repo update
+```
+
+Next install the custom resource definitions:
+
+```bash
+helm install aiven-operator-crds aiven/aiven-operator-crds
+```
+
+Finally install the operator:
```

Comment:
Installing the Operator without disabling the `webhooks` generates the following error:

```
$ helm install aiven-operator aiven/aiven-operator

Error: template: aiven-operator/templates/ensure_cert_manager_if_webhooks_are_enabled.yaml:4:8: executing "aiven-operator/templates/ensure_cert_manager_if_webhooks_are_enabled.yaml" at <fail "Required Cert Manager CRDs are missing even though Webhooks are enabled and Cert Manager is required">: error calling fail: Required Cert Manager CRDs are missing even though Webhooks are enabled and Cert Manager is required
```

I recommend:
1. Add instructions to install cert-manager [via Helm](https://cert-manager.io/docs/installation/helm/) (and verify the installation) _before_ the installation, informing the user why cert-manager is required. We can also add a `>` block telling the user the possibility of disabling the webhooks.

Reply:
added instructions on cert-manager to the prerequisites page